trl/trainer/grpo_config.py

```diff
@@ -111,8 +111,10 @@ class GRPOConfig(TrainingArguments):
             speed, but may be numerically unstable for long training runs.
         num_iterations (`int`, *optional*, defaults to `1`):
             Number of iterations per batch (denoted as μ in the algorithm).
-        epsilon (`float`, *optional*, defaults to `0.2`):
-            Epsilon value for clipping.
+        epsilon_low (`float`, *optional*, defaults to `0.2`):
+            Lower-bound epsilon value for clipping.
+        epsilon_high (`float`, *optional*, defaults to `0.28`):
+            Upper-bound epsilon value for clipping.
```

Comment:
This change isn't backward compatible


What about this instead:

```suggestion
        epsilon (`float`, *optional*, defaults to `0.2`):
            Epsilon value for clipping.
        epsilon_high (`float`, *optional*, defaults to `None`):
            Upper-bound epsilon value for clipping. If not specified, the upper bound is the same as the lower bound specified in argument `epsilon`. Paper []() recommends `0.28`.
```

Reply:
then we need to keep the `epsilon` in the main grpo trainer ?

Reply:
I guess